lib/prometheus_exporter_port.rb

```diff
@@ -24,6 +27,6 @@ def self.call
   end
 
   def self.tenant_mode
-    ENV.fetch('TENANT_MODE', 'multitenant')
+    ENV['TENANT_MODE'].to_s.empty? ? 'multitenant' : ENV['TENANT_MODE']
```

Comment:
```suggestion
    ENV['TENANT_MODE'].presence || 'multitenant'
```

Reply:
We can't, it is used before activesupport is loaded